dags.yaml

```diff
@@ -281,6 +300,7 @@ bqetl_stripe:
     retry_delay: 5m
 
 bqetl_org_mozilla_fenix_derived:
+  schedule_interval: 0 2 * * *
```

Comment:
cc @acmiyaguchi on this scheduling change. This DAG has no hope of running until 02:00 since it relies on copy_deduplicate which runs at 01:00 and takes an hour.